sdk/src/azure/core/az_mqtt5_rpc_server_hfsm.c

```diff
@@ -424,20 +439,47 @@ static az_result waiting(az_event_policy* me, az_event event)
  */
 static az_result faulted(az_event_policy* me, az_event event)
 {
-  az_result ret = AZ_ERROR_HFSM_INVALID_STATE;
-  (void)me;
-#ifdef AZ_NO_LOGGING
-  (void)event;
-#endif // AZ_NO_LOGGING
+  az_result ret = AZ_OK;
+  az_mqtt5_rpc_server* this_policy = (az_mqtt5_rpc_server*)me;
 
   if (_az_LOG_SHOULD_WRITE(event.type))
   {
     _az_LOG_WRITE(event.type, AZ_SPAN_FROM_STR("az_rpc_server/faulted"));
   }
+  switch (event.type)
+  {
+    case AZ_HFSM_EVENT_ENTRY:
+    {
+      // if ((az_event_policy*)this_policy->inbound_policy != NULL)
```

Comment:
cod still needed?

Reply:
This will get uncommented in Raul's relay PR, but I can go ahead and remove it for now!